charts/atlantis/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: v0.21.0
+appVersion: v0.22.0
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.9.2
+version: 4.9.3
 keywords:
```

Comment:
The app version should bump the minor version to keep it consistent with previous helm chart releases. We should do this in a separate pr

Reply:
sure, will remove it

Reply:
Let's just bump the minor version since the changes youre doing are still present, no?

Ref https://github.com/runatlantis/helm-charts/pull/221/files

Reply:
I'm not sure if this will break existing functionality so I will wait for others to review this PR

Reply:
I updated the patch version, not sure if the changes in the values.yaml file require an update of the minor version.